eppoclient/value.go

```diff
@@ -64,6 +65,9 @@ func castInterfaceToValue(valueInterface interface{}) Value {
 			return Null()
 		}
 		return Bool(*v)
+	case map[string]interface{}:
+		out, _ := json.Marshal(&v)
+		return String(string(out))
```

Comment:
convert native `map[string]interface` back to a string - sad and wasteful. we'll come back to this in the future.

Reply:
I think this makes sense actually. Now or in the future we can have a `getParsedJSONAssignment()` that returns the marshalled out. And then `getJSONStringAssignment()` that basically returns `String(string(getParsedJSONAssignment(...))` or something like that. Essentially what is happening right here. So I think this is fine!